### PR TITLE
code review checklist

### PR DESCRIPTION
**Pull request by EYRAM565**
This is a reference checklist for all members when they are facing issues concerning PR.

### CODE REVIEW

**Comment by EYRAM565** [comment]
@arthurbuhl

**Comment by proful** [comment]
Use @arthurbuhl format

Please don't remove earlier content

**Comment by EYRAM565** [comment]
I didn't I just pasted mine under his.

File: docs/CODE-REVIEW-CHECKLIST.MD [hunk 1]
```diff
@@ -8,3 +8,8 @@
 - [x] Using `.md` file extension is a must.
 - [x] Please add some basic bio about yourself in the first PR.
 - [ ] For the second PR, you can update the same file and add more details.
+
+[x] You need to create your second PR from your local machine by cloning your the forked repo and saying few things about yourself in the doc folder. 
+[x] Then create your PR again from your local machine this time you have update your forked repo from the main repo. 
+[x] After that you state your experience whether you are a beginner ,intermediate or mentor in the TEAM.md file in doc folder.
+[x] if you having any difficulties in creating the second and third PR click this https://www.youtube.com/channel/UCKxUmbHq5P5pd5IyUiZ8MHA to help
```
Comment (arthurbuhl) [inline docs/CODE-REVIEW-CHECKLIST.MD:15]: Just add a "`- `" (dash and space after that) before every `[x]` to convert it into checklist.

Comment (EYRAM565) [reply docs/CODE-REVIEW-CHECKLIST.MD:15]: okay

Comment (EYRAM565) [reply docs/CODE-REVIEW-CHECKLIST.MD:15]: thanks.

Comment (EYRAM565) [reply docs/CODE-REVIEW-CHECKLIST.MD]: I have done it kindly check.

**Review by arthurbuhl — APPROVED** [review]
This looks good to me. I think this can be merged now.

**Comment by proful** [comment]
No need to give You tube links

**Review by Neaz-Mahmood — APPROVED** [review]

**Comment by Neaz-Mahmood** [comment]
good to go.

**Review by nishanknavelkar — APPROVED** [review]

**Pull request merged**